<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Staff</class>
 <widget class="QDialog" name="Staff">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>792</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Staff</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QSplitter" name="splitter_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>751</width>
     <height>351</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QSplitter" name="splitter">
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QWidget" name="">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="pushButton_logout">
            <property name="text">
             <string>Logout</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_run">
            <property name="text">
             <string>Run</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="output_note">
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="plainText">
           <string>No Query Yet</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Command</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="command"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QTableView" name="tableView_query"/>
   </widget>
   <widget class="QWidget" name="">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Queued Vacancies</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="tableView_vacancies"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="pushButton_accept">
         <property name="text">
          <string>Accept</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_decline">
         <property name="text">
          <string>Decline</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
